Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```diff
@@ -180,13 +180,16 @@
   - type: Projectile
     damage:
       types:
-        Heat: 5
+        Shock: 10
     soundHit:
       path: "/Audio/Weapons/Guns/Hits/taser_hit.ogg"
     soundForce: true
   - type: StunOnCollide
-    stunAmount: 5
-    knockdownAmount: 5
+    stunAmount: 0
+    knockdownAmount: 0
+    slowdownAmount: 4
+    walkSpeedMultiplier: 0.2
+    runSpeedMultiplier: 0.2
```

Comment:
If the shots stay at 2 this is fine, but the slowdown is too slow otherwise. Make them slower than a bola but not as slow as this.

Reply:
I did not revert the shot count, It is still at 2 and not 5.

Reply:
oh you wanted it less slow only if the shot count was not still 2, I'm reverting.